lib/ramble/ramble/cmd/results.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2022-2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import llnl.util.tty as tty
+
+from ramble.config import ConfigError
+import ramble.experimental.uploader
+
+
+description = "take actions on experiment results"
+section = "workspaces"
+level = "short"
+
+subcommands = [
+    'file_upload',
+]
+
+#: Dictionary mapping subcommand names and aliases to functions
+subcommand_functions = {}
+
+
+def results_file_upload_setup_parser(subparser):
+    subparser.add_argument(
+        'filename', metavar='filename',
+        help='path of file to upload')
+
+
+def results_file_upload(args):
+    """Imports Ramble experiment results from JSON file and uploads them."""
+    imported_results = ramble.experimental.uploader.results_file_import(args.filename)
+
+    if ramble.config.get('config:upload'):
+        # Read upload type and push it there
+        if ramble.config.get('config:upload:type') == 'BigQuery':  # TODO: enum?
+            try:
+                formatted_data = ramble.experimental.uploader.format_data(imported_results)
```

Comment:
Small nit, but given we call into the uploader multiple time, I think it's probably better if we move a lot of these upload calls outside of command (either into workspace or the uploader) 

You can see for the `analyze --upload` command we chose to do most of the logic outside the command, what do you think?

Reply:
When I wrote this I forgot we don't have an active workspace, maybe the more appropriate thing to do is to push a lot of this logic down into the uploader class?

Reply:
I think that sounds right. Going to talk to Doug about how to do that and see if we need to update the 'workspace analyze upload' logic to use the same method

Reply:
Removed unused 'workspace name' parameter from perform_upload(), which enabled upload_results() to be moved into uploader module and used for both workspace upload and results import upload.